packages/components/skeleton/src/Skeleton.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+export const skeletonStyles = cva(['flex flex-row !flex-wrap'], {
+  variants: {
+    isAnimated: {
+      true: [
+        '[mask-image:linear-gradient(90deg,#000_40%,rgba(0,0,0,.3)_60%,#000_100%)]',
+        '[mask-size:200%_100%]',
+        'animate-shimmer',
```

Comment:
You can add `motion-reduce:!animate-none` for a11y :)

Reply:
Don't we add this globally? Wasn't aware of that, thx!